controllers/main.js

```diff
@@ -1,10 +1,22 @@
+const { v4: uuidv4 } = require("uuid");
+
+const PlaySession = require("../models/session");
 const { getPixelatedImage } = require("../middleware");
 
 const {
   cardStyle,
   resultsStyle,
 } = require("../public/javascripts/extraStyles.js");
 
+module.exports.renderPlaySettings = (req, res) => {
+  res.render("./main/play-settings.ejs", {
+    playSessionId: uuidv4(),
+    extraStyles: cardStyle,
+    previewImage:
+      "https://res.cloudinary.com/dyguovdbc/image/upload/v1676908287/pixelify/placeholder-image_ykgw2b.jpg",
```

Comment:
I would recommend to store this link in a constant for cleanliness sake

Reply:
That would definitely make it looks a lot cleaner, will do